### ISSUE TITLE
SSD Mobilenet training performing worse with random crop data augmentation

### ISSUE THREAD

**Issue by funkysandman** [tensorflow/models#4162]
### System information
- **What is the top-level directory of the model you are using**: models/research
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:  Linux on google cloud
- **TensorFlow installed from (source or binary)**:  binary
- **TensorFlow version (use command below)**:1.6
- **Bazel version (if compiling from source)**:na
- **CUDA/cuDNN version**:na
- **GPU model and memory**:na
- **Exact command to reproduce**: train a dataset with / without ssd_random_crop in pipeline

### Describe the problem

Introducing the data augmentation SSD_RANDOM_CROP into my training pipeline.config is resulting in worse results than leaving it out.  My understanding was that augmenting the corpus of images with more variations would help train the model further, but so far it's made every training run worse for my dataset.  Is there something wrong with the cropping algorithm or am I missing an important detail?

I have about 90 training images and 50 test images with only 1 class (meteor), most at 1920x1200px.
Without the cropping the model converges to about .27 at 5000 steps.
With cropping, the model converges to about 2.5 at 20000 steps.

The model trained without cropping out-performs the latter.  I've tried other data augmentation options that have helped performance which include:
random_horizontal_flip
random_vertical_flip
random_rot90
random_rgb_to_gray

Any insight would be much appreciated.

### Source code / logs

I am using the following model:
http://download.tensorflow.org/models/object_detection/ssd_mobilenet_v2_coco_2018_03_29.tar.gz

Here is my pipeline.config:

model {
  ssd {
    num_classes: 1
    image_resizer {
      fixed_shape_resizer {
        height: 300
        width: 300
      }
    }
    feature_extractor {
      type: "ssd_mobilenet_v2"
      depth_multiplier: 1.0
      min_depth: 16
      conv_hyperparams {
        regularizer {
          l2_regularizer {
            weight: 3.9999998989515007e-05
          }
        }
        initializer {
          truncated_normal_initializer {
            mean: 0.0
            stddev: 0.029999999329447746
          }
        }
        activation: RELU_6
        batch_norm {
          decay: 0.9997000098228455
          center: true
          scale: true
          epsilon: 0.0010000000474974513
          train: true
        }
      }
      use_depthwise: true
    }
    box_coder {
      faster_rcnn_box_coder {
        y_scale: 10.0
        x_scale: 10.0
        height_scale: 5.0
        width_scale: 5.0
      }
    }
    matcher {
      argmax_matcher {
        matched_threshold: 0.5
        unmatched_threshold: 0.5
        ignore_thresholds: false
        negatives_lower_than_unmatched: true
        force_match_for_each_row: true
      }
    }
    similarity_calculator {
      iou_similarity {
      }
    }
    box_predictor {
      convolutional_box_predictor {
        conv_hyperparams {
          regularizer {
            l2_regularizer {
              weight: 3.9999998989515007e-05
            }
          }
          initializer {
            truncated_normal_initializer {
              mean: 0.0
              stddev: 0.029999999329447746
            }
          }
          activation: RELU_6
          batch_norm {
            decay: 0.9997000098228455
            center: true
            scale: true
            epsilon: 0.0010000000474974513
            train: true
          }
        }
        min_depth: 0
        max_depth: 0
        num_layers_before_predictor: 0
        use_dropout: false
        dropout_keep_probability: 0.800000011920929
        kernel_size: 3
        box_code_size: 4
        apply_sigmoid_to_scores: false
      }
    }
    anchor_generator {
      ssd_anchor_generator {
        num_layers: 6
        min_scale: 0.20000000298023224
        max_scale: 0.949999988079071
        aspect_ratios: 1.0
        aspect_ratios: 2.0
        aspect_ratios: 0.5
        aspect_ratios: 3.0
        aspect_ratios: 0.33329999446868896
      }
    }
    post_processing {
      batch_non_max_suppression {
        score_threshold: 0.30000001192092896
        iou_threshold: 0.6000000238418579
        max_detections_per_class: 100
        max_total_detections: 100
      }
      score_converter: SIGMOID
    }
    normalize_loss_by_num_matches: true
    loss {
      localization_loss {
        weighted_smooth_l1 {
        }
      }
      classification_loss {
        weighted_sigmoid {
        }
      }
      hard_example_miner {
        num_hard_examples: 3000
        iou_threshold: 0.9900000095367432
        loss_type: CLASSIFICATION
        max_negatives_per_positive: 3
        min_negatives_per_image: 3
      }
      classification_weight: 1.0
      localization_weight: 1.0
    }
  }
}
train_config {
  batch_size: 10
  **data_augmentation_options {
    ssd_random_crop {
    }
  }**

  data_augmentation_options {

  optimizer {
    rms_prop_optimizer {
      learning_rate {
        exponential_decay_learning_rate {
          initial_learning_rate: 0.004000000189989805
          decay_steps: 800720
          decay_factor: 0.949999988079071
        }
      }
      momentum_optimizer_value: 0.8999999761581421
      decay: 0.8999999761581421
      epsilon: 1.0
    }
  }
  fine_tune_checkpoint: "gs://funkystorage/ssd_mobilenet_v2_coco_2018_03_29/model.ckpt"
  num_steps: 20000
  fine_tune_checkpoint_type: "detection"
}
train_input_reader {
  label_map_path: "gs://funkystorage/data/object-detection.pbtxt"
  tf_record_input_reader {
    input_path: "gs://funkystorage/data/train.record"
  }
}
eval_config {
  num_examples: 8000
  max_evals: 10
  use_moving_averages: false
}
eval_input_reader {
  label_map_path: "gs://funkystorage/data/object-detection.pbtxt"
  shuffle: false
  num_readers: 1
  tf_record_input_reader {
    input_path: "gs://funkystorage/data/test.record"
  }
}

**Comment by robieta** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

If you think we've misinterpreted a bug, please comment again with a clear explanation, as well as all of the information requested in the [issue template](https://github.com/tensorflow/tensorflow/issues/new). Thanks!
